src/core/shader.js

```diff
@@ -60,9 +60,7 @@ Shader.prototype = {
     var self = this;
     var variables = {};
     var schema = this.schema;
-    var squemaKeys = Object.keys(schema);
-    squemaKeys.forEach(processSquema);
-    function processSquema (key) {
```

Comment:
IMO, adds indirection making me jump up and down to read the code

`s/sqema/schema`